br/pkg/lightning/backend/local/local.go

```diff
@@ -395,7 +395,9 @@ type BackendConfig struct {
 	ConnCompressType config.CompressionType
 	// concurrency of generateJobForRange and import(write & ingest) workers
 	WorkerConcurrency int
-	KVWriteBatchSize  int
```

Comment:
renamed to `KVWriteBatchCount`

Reply:
we'd better use a new name, rather than change the meaning of it. If we debug in different release version this is confusing

Reply:
we have this name start from 7.1, if we pick to 7.1, i think it's ok